python/dgl/nn/pytorch/conv/gatconv.py

```diff
@@ -327,6 +329,11 @@ def forward(self, graph, feat, get_attention=False):
             e = self.leaky_relu(graph.edata.pop("e"))
             # compute softmax
             graph.edata["a"] = self.attn_drop(edge_softmax(graph, e))
+            if edge_weight is not None:
+                graph.edata["a"] = (
+                    graph.edata["a"]
+                    * edge_weight.tile(self._out_feats, self._num_heads, 1).T
```

Comment:
`graph.edata["a"]` is of shape `(num_edges, num_heads, 1)`. I don't think `edge_weight.tile(self._out_feats, self._num_heads, 1).T` is right.

Reply:
It was my bad. I fixed the code and confirm that attention vector and edge_weight vector are same shape.

Reply:
Thank you. This works now.